client/src/modules/list/components/ListItem.jsx

```diff
@@ -41,16 +60,118 @@ class ListItem extends PureComponent {
     // Adding new comment will be handled in next tasks
   };
 
+  handleDataUpdate = () => {
+    const { itemDescription, link } = this.state;
+    const {
+      data: { description: previousDescription, link: previousLink }
+    } = this.props;
+    const isLinkUpdated = !areStringsEqual(previousLink, link);
+    const isDescriptionUpdated = !areStringsEqual(
```

Comment:
why not using: https://lodash.com/docs/4.17.11#isEqual? Why creating own function for comparison?